conda/conda_build_config.yaml

```diff
@@ -0,0 +1,26 @@
+# from conda-forge-pinning
```

Comment:
for me this is misleading. Why do we not refer to the conda-forge config file.
See https://conda-forge.org/docs/maintainer/pinning_deps/